packages/tpetra/core/example/Finite-Element-Assembly/typedefs.hpp

```diff
@@ -50,8 +50,10 @@
 #include <Tpetra_CrsGraph.hpp>
 #include <Tpetra_CrsMatrix.hpp>
 
-typedef int LocalOrdinal;
-typedef long long GlobalOrdinal;
+// Get LocalOrdinal & GlobalOrdinal from Map defaults.
+typedef Tpetra::Map<>::local_ordinal_type LocalOrdinal;
+typedef Tpetra::Map<>::global_ordinal_type GlobalOrdinal;
+
 typedef double Scalar;
 
 typedef Tpetra::Details::DefaultTypes::node_type NT;
```

Comment:
For the future, use Tpetra::Map<>::node_type instead of the Tpetra::Details thing.  Otherwise, this is good; thanks! :D